src/components/Submit.js

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from 'react';
```

Comment:
this file shouldn't exist. it's a js file. Also all the logic related to the user creation component should be in the the user creation tsx file